pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.poetry]
+description = "Sample code for a simple chat webapp integrating with Azure OpenAI."
+authors = ["Abigail Hartman <Abigail.Hahn@microsoft.com>"]
```

Comment:
Is there a better email address for support that we can use here, or can we add a few other people to this list?

Reply:
Fixed! Thanks